### ISSUE TITLE
No links to Dolphin's Twitter/Facebook/Youtube accounts on the main site

### ISSUE THREAD

**Issue by MaJoRoesch** [dolphin-emu/www#44]
Well... it's kind of self explanatory! The primary way social media verify accounts is by checking with the organization's website. It's mostly an automated process, even. But without any links on our mainsite, they'll never find it!

**Comment by Pringo** [comment]
Yes I completely agree with this and have written about it before: https://forums.dolphin-emu.org/Thread-dolphin-and-social-media

Personally I would like to see small icons for Twitter/Facebook/YouTube (and the Dolphin Google+ page as well if that hasn't been considered abandoned by now) somewhere in the dolphin-emu.org footer.
